README.md

```diff
@@ -2,88 +2,55 @@
 
 The Baseball Card Store Dapp sells baseball cards as NFT tokens in exchange for tokens.
 
-## Run the Dapp
-
-To run the dapp, you'll use 3 separate terminal windows
-
-### Initialize the Agoric VM
+## Initialize the Dapp project
 
 ```sh
-# Terminal 1
 cd $HOME
 # Install the dapp into a directory named demo
-npm create @agoric/dapp@getting-started-dev init --dapp-template dapp-card-store --dapp-branch mhofman/hack-getting-started demo
+npm create @agoric/dapp -- --dapp-template dapp-card-store --dapp-branch mhofman/hack-getting-started demo
```

Comment:
Why remove the `@getting-started-dev` tag? until we publish a new `latest`, don't we need it?

Reply:
I published `@agoric/create-dapp` already since it uses the `agoric@getting-started` already, and only relies on `agoric init` to do the right thing, which it does.

Reply:
The `latest` tag is technically older: https://www.npmjs.com/package/@agoric/create-dapp?activeTab=versions